### DISCUSSION TITLE
Just the "good" news, please! 📰😊

### DISCUSSION THREAD

**Discussion by chromagic-development** [chromagic-development/Machina#20] [Announcements]
I made a new AI macro that reports only the "good" news after scraping the headlines from the AP news website. Thanks go to my wife for making this request after wanting a break from hearing all the glum events happening across the world.

**Comment by AlexandreSAfonso** [comment]
Hello!

Is there any reason to organize the plugins like this in the project? Wouldn't it be interesting to separate each one, both for those who are thinking of downloading one or more and for potential contributors?

I was thinking about trying out a plugin focused on simulators. This plugin could read values ​​from an API (from the X-plane 12 simulator, in this case) and also act through this API with some commands.

**Reply by chromagic-development** [reply]
Great idea and thanks for your interest! I added each VM plugin as a separate Microsoft Visual Studio solution (e.g. the folders for AskChatGPTPlugin, GetCommandPlugin, SpeakTextPlugin, etc). and I suppose new plugins can be contributed that way. I also packaged all the project binaries to specifically run Machina in a zip file (Machina.zip) that includes the listed plugins for a ready-to-go implementation, but you can easily remove the plugins you don't want to use. You can also add custom VM macros or change existing ones in the Machina profile imported from Machina.xml to meet your X-plane simulator needs and perhaps, export it as a new profile for the X-plane simulator, like xplane.xml.

Let me know if that makes sense and if you have any other thoughts or questions.
